trino-aws-proxy-spi/src/main/java/io/trino/aws/proxy/spi/plugin/config/AssumedRoleProviderConfig.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.aws.proxy.spi.plugin.config;
+
+import io.airlift.configuration.Config;
+import jakarta.validation.constraints.NotNull;
+
+public class AssumedRoleProviderConfig
+        implements PluginIdentifierConfig
+{
+    private String identifier = "noop";
```

Comment:
I don't mean to be a pain on this, but I think this is important...

Can we remove `noop` as a default identifier and use `Optional` instead (or empty string)?

Reply:
Fair point, I was thinking about this too but I forgot about it